src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -129,5 +129,34 @@ internal static bool IsPrimitiveType(this CorElementType et)
         [Intrinsic]
         internal static bool IsKnownConstant(int t) => false;
 #pragma warning restore IDE0060
+
+#if NATIVEAOT
+        [System.Runtime.RuntimeExport("RhRuntimeHelpers_MemSet")]
+#endif
+        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
```

Comment:
```suggestion
```

Reply:
Yeah it was just a CI test, although, I forgot to change VM side to set PRECODE.
I'll remove those once https://github.com/dotnet/runtime/pull/98682 is merged (this PR is blocked on that)